### ISSUE TITLE
switching schedule on and off in the UI shows graphQL error

### ISSUE THREAD

**Issue by wj-c** [dagster-io/dagster#23567]
### Dagster version

1.8.0

### What's the issue?

after updating 1.8.0, turning on or off a schedule in the UI shows the graphQL error as following:

```bash
Operation name: {ScheduleStateQuery}

Message: Variable '$selector' of required type 'InstigationSelector!' was not provided.

Path: 

Locations: [{"line":1,"column":40}]
```

### Deployment type

None

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by lulucici** [comment]
same here

**Comment by cmpadden** [comment]
Hi @wj-c and @lulucici - thank you for reporting this.

This has been patched for Dagster+ users, and a fix will be included in the upcoming 1.8.1 release this week.
